Update register-scan-hive-metastore-source.md (#84256)
Azure/azure-content: articles/purview/register-scan-hive-metastore-source.md

       :::image type="content" source="media/register-scan-hive-metastore-source/databricks-jdbc-connection.png" alt-text="databricks-jdbc-url-details" border="true":::

       > [!NOTE]

>        > When you copy the URL from *hive-site.xml*, be sure you remove `amp;` from the string or the scan will fail. To this URL, append the path to the location where the SSL certificate is placed on your VM. You can download the SSL certificate here : [https://www.digicert.com/CACerts/BaltimoreCyberTrustRoot.crt.pem](https://www.digicert.com/CACerts/BaltimoreCyberTrustRoot.crt.pem). 

Reviewer: ```suggestion
       > When you copy the URL from *hive-site.xml*, remove `amp;` from the string or the scan will fail. Then append the path to your SSL certificate to the URL. This will be the path to the SSL certificate's location on your VM. You can download the SSL certificate here : [https://www.digicert.com/CACerts/BaltimoreCyberTrustRoot.crt.pem](https://www.digicert.com/CACerts/BaltimoreCyberTrustRoot.crt.pem). 
```

>        > When entering the local file system paths in the Purview studio scan configuration screen, rmember to change the Windows path separator character from `\` to `/`. For example: If your MariaDB JAR file is *C:\mariadb-jdbc.jar*, change it to *C:/mariadb-jdbc.jar*. Make the same change to the Metastore JDBC URL `sslCA` parameter: For example, if it is placed at local file system path *D:\Drivers\SSLCert\BaltimoreCyberTrustRoot.crt.pem*, change it to *D:/Drivers/SSLCert/BaltimoreCyberTrustRoot.crt.pem*.

Reviewer: ```suggestion
       > When entering local file system paths in the Purview studio scan configuration screen, remember to change the Windows path separator character from `\` to `/`. For example: If your MariaDB JAR file is *C:\mariadb-jdbc.jar*, change it to *C:/mariadb-jdbc.jar*. Make the same change to the Metastore JDBC URL `sslCA` parameter: For example, if it is placed at local file system path *D:\Drivers\SSLCert\BaltimoreCyberTrustRoot.crt.pem*, change it to *D:/Drivers/SSLCert/BaltimoreCyberTrustRoot.crt.pem*.
```
